shopfloor/services/location_content_transfer.py

```diff
@@ -767,9 +767,13 @@ def postpone_package(self, location_id, package_level_id):
         package_level = self.env["stock.package_level"].browse(package_level_id)
         if not location.exists():
             return self._response_for_start(message=self.msg_store.record_not_found())
-        if package_level.exists():
-            package_level.shopfloor_postponed = True
         move_lines = self._find_transfer_move_lines(location)
+        if package_level.exists():
+            pickings = move_lines.mapped("picking_id")
+            sorter = self.actions_for("location_content_transfer.sorter")
+            sorter.feed_pickings(pickings)
+            package_levels = sorter.package_levels()
+            package_level.shopfloor_postpone(package_levels)
```

Comment:
if we have move lines + package levels in a location, it should postpone at max(max(lines), max(package_levels)) I think

Reply:
@sebalix ?

Reply:
Indeed, if we are processing move lines and package levels from the same scope, I had to do it this way. Thanks!

Reply:
@simahawk check my last fixup!

Reply:
neat :)